.github/workflows/spmd_tests.yaml

```diff
@@ -39,3 +39,71 @@ jobs:
       - name: Test with pytest
         run: |
           pytest --shard-id=${{ matrix.shard }} --num-shards=4 --cov=spmd test/spmd/
+
+  pytest_tests_gpu:
+    runs-on: linux.16xlarge.nvidia.gpu
+    strategy:
+      matrix:
+        shard: ["0", "1", "2", "3"]
+    env:
+      DOCKER_IMAGE: gingerhugo/cuda-11.3-python-3.9:v1.0.1
```

Comment:
Should we point this to the dockerfile you added in this PR?

Reply:
This is indeed the docker image built from the Dockerfile in this PR. This is the repo I created on Docker hub, lol.. It was a wired name because I just created them a while back when I was still at school.